variables.tf

```diff
@@ -3,3 +3,5 @@ variable "vpc_cidr" {}
 variable "public_subnet_cidr" {}
 
 variable "private_subnet_cidr" {}
+
+variable "env_code" {}
```

Comment:
could you please, add a newline at the end of file.

Reply:
ok thanks, done